### PR TITLE
Loss format from .3f to .3g in the tqdm

### LINKED ISSUE #4971: Loss format from .3f to .3g in the training loop
## 🚀 Feature

I propose to change the default format of loss during training (in the tqdm bar) from .3f to .3g

### Motivation

When using pytorch-lightning with losses that are quite close to zero, the tqdm information during training becomes non informative, because the loss is always 0.000. For example:
```
Epoch 41:  76%|██████                                   | 37/49 [00:00<00:00, 65.74it/s, loss=0.000, v_num=92]
```
The proposed change takes it to 
```
Epoch 41:  76%|██████                                   | 37/49 [00:00<00:00, 65.74it/s, loss=2.2e-08, v_num=92]
```
This change from .3f to .3g has also an advantage when loss is a large number
```
$print('loss: %.3f' % 80808423243)
loss: 80808423243.000
$print('loss: %.3g' % 80808423243)
loss: 8.08e+10
```
In other situations, the output of .3f and .3g does not change
```
$print('loss: %.3f' % .884)
loss: 0.884
$print('loss: %.3g' % .884)
loss: 0.884
```

### PR DESCRIPTION
**Pull request by fjhheras**
## What does this PR do?

When using pytorch-lightning with losses that are quite close to zero, the tqdm information during training becomes non informative, because the loss is always shown as 0.000. For example:

```
Epoch 41:  76%|██████                                   | 37/49 [00:00<00:00, 65.74it/s, loss=0.000, v_num=92]
```
The proposed change takes it to 
```
Epoch 41:  76%|██████                                   | 37/49 [00:00<00:00, 65.74it/s, loss=2.2e-08, v_num=92]
```

Fixes #4971

## Before submitting

I am unsure about whether the ones I did not tick were necessary, but I am happy to modify test/docs/CHANGELOG if requested.

- [X] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [X] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [X] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? 
- [X] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:  

 - [X] Is this pull request ready for review? (if not, please submit in draft mode)
 - [X] Check that all items from **Before submitting** are resolved
 - [X] Make sure the title is self-explanatory and the description concisely explains the PR
 - [X] Add labels and milestones (and optionally projects) to the PR so it can be classified; _Bugfixes should be including in bug-fix release milestones (m.f.X) and features should be included in (m.X.b) releases._
 

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by Borda — APPROVED** [review]
nice suggestion :]

**Review by carmocca — APPROVED** [review]

**Review by ydcjeff — APPROVED** [review]

**Review by rohitgr7 — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Comment by Borda** [comment]
@williamFalcon mind review :]

**Review by s-rog — APPROVED** [review]
